src/analysis/retail/druid/restoration/modules/spells/Innervate.tsx

```diff
@@ -108,14 +125,24 @@ class Innervate extends Analyzer {
         <strong>Per-Cast Breakdown</strong>
         <small> - click to expand</small>
         {this.castTrackers.map((cast, ix) => {
-          const targetName = cast.targetId === undefined ? 'SELF' : 'ALLY';
-          const metThresholdMana = cast.manaSaved >= INNERVATE_MANA_REQUIRED;
-          const castWildGrowth =
-            cast.casts.filter((c) => c.ability.guid === SPELLS.WILD_GROWTH.id).length > 0;
-          const overallPerf =
-            metThresholdMana && castWildGrowth
-              ? QualitativePerformance.Good
-              : QualitativePerformance.Fail;
+          const sourceName = cast.sourceId === undefined ? 'SELF' : 'EXTERNAL';
+          const endTime = Math.min(this.owner.fight.end_time, cast.timestamp + 8_000);
+          const activeRampSpellTime = this.alwaysBeCasting.getActiveTimeMillisecondsFiltered(
```

Comment:
i like this. if performance is bad then IMHO the correct thing to do is to work on improving `ABC` internals, not cutting the feature.